ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```
@@ -117,16 +117,16 @@ library
     , cardano-crypto
     , cardano-crypto-class
     , cardano-crypto-wrapper
-    , cardano-ledger-allegra        ^>=1.2.5
-    , cardano-ledger-alonzo         ^>=1.5.1
-    , cardano-ledger-api            ^>=1.7
-    , cardano-ledger-babbage        ^>=1.5.1
-    , cardano-ledger-binary         ^>=1.2.1
-    , cardano-ledger-byron          ^>=1.0.0.3
-    , cardano-ledger-conway         ^>=1.11
-    , cardano-ledger-core           ^>=1.9
-    , cardano-ledger-mary           ^>=1.4
-    , cardano-ledger-shelley        ^>=1.8
+    , cardano-ledger-allegra        ^>=1.3
+    , cardano-ledger-alonzo         ^>=1.6
+    , cardano-ledger-api            ^>=1.8
+    , cardano-ledger-babbage        ^>=1.6
+    , cardano-ledger-binary         ^>=1.3
+    , cardano-ledger-byron          ^>=1.0.0.4
```

Patch versions should not appear in cabal bounds (previous `^>=1.0.0.3` was wrong too)
```suggestion
    , cardano-ledger-byron          ^>=1.0
```

---

Fixed